Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.py

```diff
@@ -0,0 +1,1082 @@
+    string += (
+        f"_Last updated {comment['lastUpdatedTime']}_\n"
+        if comment["lastUpdatedTime"]
+        else ""
+    )
+    string += f"{comment['comment']}\n\n"
+    string += f"_id: {comment['id']}_\n"
+    string += f"_Flags: {str(comment['flags'])}_\n" if comment["flags"] else ""
+    string += "* * *\n"
+    return string
+
+
+def pretty_print_comments(comments: list, title: str = None) -> str:
+    string = title if title else ""
+    for comment in comments:
+        string += pretty_print_comment(comment)
+    return string
+
+
+def pretty_print_events(result: dict, title: str = None) -> str:
+    string = title if title else ""
+    string += "_Count: {}, showing {} events, from {} to {}_\n".format(
+        result["count"], result["size"], result["offset"], result["limit"]
+    )
+    string += tableToMarkdown("Events", result["data"])
+    return string
+
+
+""" COMMAND FUNCTIONS """
+
+
+def test_module_command() -> str:
+    response = get_current_user()
+    if response["responseCode"] == 200:
+        return "ok"
+    return (
+        f"Unable to communicate with Argus API {response['responseCode']}, {response}"
+    )
+
+
+def fetch_incidents(last_run: dict, first_fetch_period: str):
+    start_timestamp = last_run.get("start_time", None) if last_run else None
+    # noinspection PyTypeChecker
+    result = advanced_case_search(
+        startTimestamp=start_timestamp if start_timestamp else first_fetch_period,
+        endTimestamp="now",
+        limit=demisto.params().get("max_limit", 0),
+        sortBy=["createdTimestamp"],
+        priority=build_argus_priority_from_min_severity(
+            demisto.params().get("min_severity", "medium")
+        ),
+        subCriteria=[
+            {"exclude": True, "status": ["closed"]},
+        ],
+        timeFieldStrategy=["createdTimestamp"],
+    )
+    incidents = []
+    for case in result["data"]:
+        incidents.append(
+            {
+                "name": f"#{case['id']}: {case['subject']}",
+                "occurred": case["createdTime"],
+                "severity": argus_priority_to_demisto_severity(case["priority"]),
+                "status": argus_status_to_demisto_status(case["status"]),
+                "details": case["description"],
+                "customFields": {
+                    "argus_id": str(case["id"]),
+                    "type": case["type"],
+                    "category": case["category"]["name"] if case["category"] else None,
+                    "service": case["service"]["name"],
+                    "lastUpdatedTime": case["lastUpdatedTime"],
+                    "createdTimestamp": case["createdTimestamp"],
+                    "customer": case["customer"]["shortName"],
+                },
+                "rawJson": json.dumps(case),
+            }
+        )
+    if result["data"]:
+        last_run["start_time"] = result["data"][-1]["createdTimestamp"] + 1
```

Comment:
May I ask why? Epoch milliseconds is what the API expects, and we reduce complexity by converting back and forth (and possible timezone issues). 
If there is no technical limitation or it will break something else, then I'd like to keep it as a timestamp. What do you think?

Reply:
Data is stored in the DB as a string, so in order to prevent data loss or problems with the conversion we send it as a string (as described in https://xsoar.pan.dev/docs/integrations/fetching-incidents#setting-last-run)

Reply:
Would it then be possible to simply store the timestamp as a string, or does it expect the "datestring" format?